websites/T/Threads/presence.ts

```diff
@@ -0,0 +1,83 @@
+const presence = new Presence({
+		clientId: "1237798561739968513",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	capitalize = (text: string): string => {
+		return text
+			.replace(/[[{(_)}\]]/g, " ")
+			.replace(/^\w/, c => c.toUpperCase());
+	};
+
+presence.on("UpdateData", async () => {
+	const [privacy, time] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("time"),
+		]),
+		{ pathname, href, search } = document.location,
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/qR37ayt.png",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).